internal/repository/repository.go

```diff
@@ -31,6 +32,8 @@ import (
 	"github.com/retr0h/gilt/v2/pkg/config"
 )
 
+const ORIGIN = "gilt"
```

Comment:
This **COULD** come from the configs, but I think leaving it as an immutable implementation detail is the safer bet.